kubetest2-gke/deployer/deployer.go

```diff
@@ -182,19 +193,25 @@ var _ types.NewDeployer = New
 func bindFlags(d *deployer) *pflag.FlagSet {
 	flags := pflag.NewFlagSet(Name, pflag.ContinueOnError)
 
-	flags.StringVar(&d.cluster, "cluster-name", "", "Cluster name. Must be set.")
+	flags.StringSliceVar(&d.clusters, "cluster-name", []string{}, "Cluster names separated by comma. Must be set. "+
+		"For multi-project profile, it should be in the format of clusterA:1,clusterB:1,clusterC:2, where the index means the index of the project.")
```

Comment:
is it 0-based or 1-based?

Reply:
It's 0-based, I have changed the example to have a project index as 0 to make it less confusing.